pkg/planner/cascades/base/base_test.go

```diff
@@ -0,0 +1,99 @@
+// Copyright 2024 PingCAP, Inc.
```

Comment:
Can we add this bench into daily-bench？ so that we can measure the benchmark result.

Reply:
removed after discussion with elsa, it not about SQL optimization cycle, just a based func, it shouldn't be changed from a long term